explanations/no-python-tests.md

```diff
@@ -0,0 +1,13 @@
+The build log of the package contained the string "Ran 0 tests in 0.000s"
```

Comment:
Since we are formatting now, could you format this with `nix shell -f channel:nixos-unstable nodePackages.prettier -c prettier --prose-wrap never --check explanations/`?